packages/augur-ui/src/modules/create-market/components/form.tsx

```diff
@@ -96,40 +97,27 @@ export default class Form extends React.Component<
     } = this.props;
     const s = this.state;
 
+    const content = CUSTOM_CONTENT_PAGES[newMarket.currentStep];
```

Comment:
since we don't actually use content directly, we should deconstruct this:

` const { mainContent, explainerBlockTitle, explainerBlockSubtexts, <etc> } = CUSTOM_CONTENT_PAGES[newMarket.currentStep];`